.github/workflows/mac_python_matrix_test.yml

```diff
@@ -35,6 +35,27 @@ jobs:
         echo "Install Pyomo dependencies..."
         echo ""
         pip install cython numpy scipy ipython openpyxl sympy pyyaml pyodbc networkx xlrd pandas matplotlib dill seaborn pymysql pyro4 pint pathos
+        echo ""
+        echo "Install GAMS..."
+        echo ""
+        wget -q https://d37drm4t2jghv5.cloudfront.net/distributions/29.1.0/macosx/osx_x64_64_sfx.exe
+        chmod +x osx_x64_64_sfx.exe
+        ./osx_x64_64_sfx.exe -q -d gams
+        cd gams/gams29.1_osx_x64_64_sfx/apifiles/Python/
+        py_ver="$(python --version)"
+        if [[ "$py_ver" == *"2.7"* ]]; then
```

Comment:
This is an edge case, but one that may show up... 3.2.7 will match the above.  A better test might be:
```
if `python -c 'import sys;sys.exit(0 if sys.version_info[:2] == (2,7) else 1)'`; then
```
For the others you can do `sys.version_info[:2] == (3,6)` and `sys.version_info[:2] >= (3,7)`.

Reply:
I could change it to match how I did the Windows ones - those checks are based off the python-version matrix value.

Reply:
I think either would be OK.  The advantage of the relational operator is we won't have to change it for future releases.  You could do something like this as well:
```
py_ver=`python -c 'import sys;print(''.join(sys.version_info[:2]))'`
gams_ver=api
for ver in api_*; do
    if test ${ver:4} -lt $py_ver; then
        gams_ver=$ver
    fi
done
cd $gams_ver
python setup.py -q install -noCheck
```

Reply:
Changes made and passed.